cmake/templates/OpenCVConfig-version.cmake.in

```diff
@@ -8,7 +8,3 @@ if(PACKAGE_FIND_VERSION VERSION_EQUAL PACKAGE_VERSION)
   set(PACKAGE_VERSION_EXACT True)
   set(PACKAGE_VERSION_COMPATIBLE True)
 endif()
-
-if(PACKAGE_FIND_VERSION VERSION_LESS PACKAGE_VERSION)
```

Comment:
Hi Vincent!

CMake documentation says:
```
PACKAGE_FIND_VERSION_MAJOR = major version if requested, else 0
```

so I believe `VERSION_EQUAL` is not required here, because it is a single number and `EQUAL` should work fine:

```
if(<variable|string> EQUAL <variable|string>)
True if the given string or variable’s value is a valid number and equal to that on the right.
```

BTW, I'm not sure, should we handle case of `PACKAGE_FIND_VERSION_MAJOR=0` here or not?